examples/buildpacks/README.md

```diff
@@ -28,11 +29,54 @@ tkn tr describe --last -o json | jq -r '.metadata.annotations["chains.tekton.dev
 # Should output "true"
 
 # Double check that the attestation and the signature were uploaded to the OCI.
-crane ls ${DOCKER_IMG}
+crane ls "${DOCKER_IMG}"
 
 # Verify the image and the attestation.
-cosign verify --key $COSIGN_PUB ${DOCKER_IMG}
-cosign verify-attestation --key $COSIGN_PUB ${DOCKER_IMG}
+cosign verify --key "$COSIGN_PUB" "${DOCKER_IMG}"
+cosign verify-attestation --key "$COSIGN_PUB" "${DOCKER_IMG}"
+```
+
+## How to run using the minikube registry
+
+This example will push the built artifact into the internal minikube registry created
+by `minikube addons enable registry`.
+
+```bash
+# Only if a cluster is needed.
+make setup-minikube
+
+# Setup tekton w/ chains
+make setup-tekton-chains tekton-generate-keys
+export COSIGN_PUB=${PWD}/cosign.pub
+
+# In a separate terminal, port forward the registry
+# ./platform/05-minikube-registry-proxy.sh
+
+# Set the registry for use in buildpacks.sh
+export REGISTRY=registry.kube-system.svc.cluster.local
+
+# Run a new pipeline.
+./examples/buildpacks/buildpacks.sh
+# Or re-run the last one.
+# tkn pipeline start buildpacks -L
+
+# Export the value of APP_IMAGE from the pipelinerun describe as DOCKER_IMG:
+APP_IMAGE=$(tkn pr describe --last -o json | jq -r '.spec.params[] | select(.name=="APP_IMAGE") | .value')
+export DOCKER_IMG=localhost:8888${APP_IMAGE#"$REGISTRY"}
+
+# Wait until it completes.
+tkn pr logs --last -f
+
+# Ensure it has been signed.
+tkn tr describe --last -o json | jq -r '.metadata.annotations["chains.tekton.dev/signed"]'
+# Should output "true"
+
+# Double check that the attestation and the signature were uploaded to the OCI.
+crane ls "${DOCKER_IMG}"
+
+# Verify the image and the attestation.
+cosign verify --key "$COSIGN_PUB" "${DOCKER_IMG}"
+cosign verify-attestation --key "$COSIGN_PUB" "${DOCKER_IMG}"
```

Comment:
This seems to duplicate a lot of what is written in the previous section. Could we just add the explanation to launch the proxy instead?

Reply:
Maybe. I was trying to reduce the amount of comments the user needed to read while walking through the example script.